CMakeLists.txt

```diff
@@ -113,7 +113,7 @@ else( WIN32 ) # Apple AND Linux
     if( APPLE )
         # Apple Specific Options Here
         message( STATUS "Configuring Eos on OS X" )
-        set( CMAKE_CXX_FLAGS "${CMAKE_C_FLAGS} -Wall -Wno-deprecated-declarations" )
+        set( CMAKE_CXX_FLAGS "${CMAKE_C_FLAGS} -std=c++14 -Wall -Wno-invalid-partial-specialization -Wno-deprecated-declarations" )
```

Comment:
It is not. I have verified that at this point CMAKE_CXX_STANDARD is indeed 14, but if I don't add the explicit --std=c++14, no joy comes to compile town.